src/Swashbuckle.AspNetCore.Newtonsoft/SchemaGenerator/NewtonsoftDataContractResolver.cs

```diff
@@ -207,7 +207,10 @@ private IEnumerable<DataProperty> GetDataPropertiesFor(JsonObjectContract jsonOb
             [ typeof(DateTimeOffset) ] = Tuple.Create(DataType.String, "date-time"),
             [ typeof(Guid) ] = Tuple.Create(DataType.String, "uuid"),
             [ typeof(Uri) ] = Tuple.Create(DataType.String, "uri"),
-            [ typeof(TimeSpan) ] = Tuple.Create(DataType.String, "date-span")
+            [ typeof(TimeSpan) ] = Tuple.Create(DataType.String, "date-span"),
+#if NET6_0_OR_GREATER
+            [ typeof(DateOnly) ] = Tuple.Create(DataType.String, "date")
```

Comment:
What about `TimeOnly`?

Reply:
AFAIK, OpenAPI/JSON schema doesn't have support for time only schema. We could get away with doing a string type and providing a pattern for the format but that seemed like a little more effort than it was worth given the resolver doesn't seem to be set up for this.

Reply:
I'm mainly concerned with how Swagger displays `TimeOnly` parameters. It should do it as a single field (since it's a single field under the hood)

![image](https://user-images.githubusercontent.com/6609929/187956278-0b9e5ca4-0efb-4f4d-ab82-f9646cfabea5.png)

Instead, currently it displays it as multiple fields, which makes it somewhat difficult to use:

![image](https://user-images.githubusercontent.com/6609929/187956504-87cad2d0-84b7-4205-98b5-909c5317b097.png)

Not sure that ASP.NET Core 7 will be able to bind `string` to `TimeOnly` out-of-the-box (since https://github.com/dotnet/aspnetcore/issues/34591#issuecomment-1225997439 got moved to ASP.NET Core 8), but same applies to `DateOnly`. If it won't be able to do it for `TimeOnly`, it won't be able to do it for `DateOnly` as well.

But NSwag resolves this dynamically somehow. If it sees `TypeConverter` to `string`, it automatically switches to displaying those types as one field.

Reply:
> Not sure that ASP.NET Core 7 will be able to bind string to TimeOnly out-of-the-box (since https://github.com/dotnet/aspnetcore/issues/34591#issuecomment-1225997439 got moved to ASP.NET Core 8), but same applies to DateOnly. If it won't be able to do it for TimeOnly, it won't be able to do it for DateOnly as well.

At the moment, minimal APIs is the only framework implementation that supports `TimeOnly`/`DateOnly` in routing. Although what the framework does is fairly independently from what Swashbuckle/NSwag does. We could totally add `TimeOnly` support here, we'd just have to figure out what schema to write. It looks like NSwag is using a "time" format for the string.

https://github.com/RicoSuter/NJsonSchema/blob/a1053c83dd798defdd0a83f98f379cadadcb1e64/src/NJsonSchema/Generation/DefaultReflectionService.cs#L151-L155

Reply:
I also used `time` in the NuGet: https://github.com/maxkoshevoi/DateOnlyTimeOnly.AspNet/blob/d239472270aaac83196a2ce1d701e60e3af670b9/DateOnlyTimeOnly.AspNet.Swashbuckle/Extensions/SwaggerGenOptionsExtensions.cs#L18

Seems like a logical name given `date-time` and `date` exists

Reply:
Added support for `TimeOnly`.